src/pages/notice/NoticeList.tsx

```diff
@@ -1,3 +1,204 @@
+> = {
+  마감임박순: 'time',
+  시급많은순: 'pay',
+  시간적은순: 'hour',
+  가나다순: 'shop',
+};
+
+// 필터가 몇 개 적용되었는지 계산해서 뱃지 등에 표시해줌
+function countAppliedFilters(filterValues: FilterValues): number {
+  let count = 0;
+  if (Array.isArray(filterValues.address) && filterValues.address.length > 0)
+    count += filterValues.address.length;
+  if (filterValues.startsAt) count += 1;
+  if (filterValues.hourlyPay) count += 1;
+  return count;
+}
+
+// ================== 컴포넌트 =========================
+export default function NoticeList({ search = '' }: NoticeListProps) {
+  const [allNotices, setAllNotices] = useState<NoticeWithShopItem[]>([]); // 현재 페이지에 노출할 공고 목록
+  const [totalCount, setTotalCount] = useState(0); // 전체 공고 수
+  const [sort, setSort] = useState<(typeof SORT_OPTIONS)[number]>(
+    SORT_OPTIONS[0],
+  ); //정렬(드롭다운값)
+  const [filterValues, setFilterValues] = useState<FilterValues>({}); // 상세필터 상태값
+  const [currentPage, setCurrentPage] = useState<number>(1); // 현재 페이지(페이지네이션)
+  const [filterOpen, setFilterOpen] = useState<boolean>(false); // 필터 모달 오픈
+  const [loading, setLoading] = useState(false); // 로딩 에러처리
+  const [error, setError] = useState<string | null>(null); // 로딩 에러처리
+
+  useEffect(() => {
+    setLoading(true);
+    setError(null);
+
+    getNotices({
+      offset: (currentPage - 1) * ITEMS_PER_PAGE,
+      limit: ITEMS_PER_PAGE,
+      address: filterValues.address?.[0],
+      keyword: search,
+      startsAtGte: filterValues.startsAt ?? undefined,
+      hourlyPayGte: filterValues.hourlyPay ?? undefined,
+      sort: sortMap[sort] ?? undefined,
+    })
+      .then((data) => {
+        setAllNotices(data.items.map((item) => item.item));
+        setTotalCount(data.count);
+      })
+      .catch((err) => setError(err.message))
+      .finally(() => setLoading(false));
+  }, [search, sort, filterValues, currentPage]);
+
+  const totalPages = Math.ceil(totalCount / ITEMS_PER_PAGE); // 페이지네이션
+  const appliedFilterCount = countAppliedFilters(filterValues); // 필터 적용
+
+  // 필터 적용
+  const handleApplyFilter = (values: FilterValues) => {
+    setFilterValues(values);
+    setCurrentPage(1);
+  };
+
+  // ===== 렌더 =====
+  if (loading)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-black">
+        잠시만 기다려주세요
+      </div>
+    );
+  if (error)
+    return (
+      <div className="flex h-500 items-center justify-center text-h3 text-red-500">
+        {error}
+      </div>
+    );
+
+  return (
+    <main>
+      {/* 맞춤 공고 */}
+      {!search && (
+        <article className="bg-red-10 px-32 py-60">
+          <section className="mx-auto flex max-w-964 flex-col">
+            <h2 className="mb-32 text-h1 font-bold">맞춤 공고</h2>
+            <div className="flex gap-14 overflow-hidden">
+              {allNotices.slice(0, 3).map((notice) => (
+                <Link
+                  key={notice.id}
+                  to={`/shops/${notice.shop.item.id}/notices/${notice.id}`}
```

Comment:
```suggestion
                  to={`/${notice.id}`}
```
❗ 공고 상세 페이지의 주소는 그냥 `/:id` 입니다~ 프로젝트 정리 문서 참고하시면 좋을 것 같아요

Reply:
네 알겠습니다 감사합니다